src/index.njk

```diff
@@ -24,27 +24,40 @@ layout: default
 
 <!-- Secondary Content -->
 <div class="w-full ff-bg-light px-6 pt-12 pb-32 md:pt-32 md:px-0">
-    <div class="container m-auto text-center max-w-2xl">
-        <div class="mt-24 w-full max-w-2xl text-center">
-            <h2>FlowForge Cloud</h2>
-            <iframe class="ff-video m-auto mt-8 w-full h-80 sm:h-96" src="https://www.youtube.com/embed/YYZDx8n17Ys">
-            </iframe>
+    <div class="container m-auto text-center max-w-4xl">
+        <div class="m-auto mt-24 w-full text-center">
+            <h2>3,000+ Integrations Available</h2>
         </div>
-        <div class=" mt-12 w-full max-w-2xl text-center flex gap-2 justify-between">
-            <div class="w-1/4 css-full">
-                <img src="./images/pictograms/security_red.png"/>
-                <h5 class="flex justify-center">SECURE</h5>
-                <label>Role-based access control at team and project levels.</label>
+        <div class="sm:mt-10 md:mt-16 w-full max-w-4xl text-center flex flex-wrap gap-2 justify-around">
+            <div class="w-full px-3 mt-4 sm:w-72 md:mt-0 css-full">
+                <img class="w-40 m-auto" src="./images/pictograms/community_red.png"/>
+                <h5 class="flex justify-center mb-2">SOCIAL & COMMS</h5>
+                <label class="font-light">Twitter, Twilio, Telegram, Slack, WhatsApp, Discord, etc.</label>
+            </div>
+            <div class="w-full px-3 mt-4 sm:w-72 md:mt-0">
+                <img class="w-40 m-auto" src="./images/pictograms/edge_red.png"/>
+                <h5 class="flex justify-center mb-2">HARDWARE</h5>
+                <label class="font-light">Servers, PLCs, Network Switches & Routers, Edge Devices, etc.</label>
+            </div>
+            <div class="w-full px-3 mt-4 sm:w-72 md:mt-0">
+                <img class="w-40 m-auto" src="./images/pictograms/candor_red.png"/>
+                <h5 class="flex justify-center mb-2">PROTOCOLS</h5>
+                <label class="font-light">WebSockets, HTTP, MQTT, AMQP, SNMP, RSS, FTP, etc.</label>
+            </div>
+            <div class="w-full px-3 mt-4 sm:w-72 lg:mt-10">
+                <img class="w-40 m-auto" src="./images/pictograms/cloud_red.png"/>
+                <h5 class="flex justify-center mb-2">CLOUD</h5>
+                <label class="font-light">AWS, Azure, S3, Google Cloup Platform, IBM Cloud, etc.</label>
```

Comment:
@joepavitt There is a typo. Google Cloup Platform -> Google Cloud Platform

Reply:
Fixed via #136